src/vm/exceptionhandling.cpp

```diff
@@ -2892,7 +2906,7 @@ CLRUnwindStatus ExceptionTracker::ProcessManagedCallFrame(
                                 {
                                     // CallHandler expects to be in COOP mode.
                                     GCX_COOP();
-                                    dwResult = CallHandler(dwFilterStartPC, sf, &EHClause, pMD, Filter X86_ARG(pCurRegDisplay->pCallerContext) ARM_ARG(pCurRegDisplay->pCallerContext) ARM64_ARG(pCurRegDisplay->pCallerContext));
+                                    dwResult = CallHandler(dwFilterStartPC, sf, &EHClause, pMD, Filter X86_ARG(pContext) ARM_ARG(pContext) ARM64_ARG(pContext));
```

Comment:
I introduced this change to isolate all the changes related with ``USE_CURRENT_CONTEXT_IN_FILTER``. 

The former form requires us to take care of this line when there are changes related with ``USE_CURRENT_CONTEXT_IN_FILTER``.

I'll update PR if you prefer the former form, but could you please take a look of this issue?